mobsf/DynamicAnalyzer/views/android/dynamic_analyzer.py

```diff
@@ -46,14 +48,17 @@ def dynamic_analysis(request, api=False):
         and_sdk = None
         apks = StaticAnalyzerAndroid.objects.filter(
             APP_TYPE='apk')
+
         for apk in reversed(apks):
+            app_dir = os.path.join(settings.UPLD_DIR, apk.MD5 + '/')
```

Comment:
Minor nit:
Can we start using pathlib for new code?
```
from pathlib import Path
app_dir = Path(settings.UPLD_DIR) / apk.MD5
```
Otherwise, we could ignore the `+ '/'` I think.